osu.Game/Overlays/OptionsOverlay.cs

```diff
@@ -52,20 +48,23 @@ public OptionsOverlay()
                 new MaintenanceSection(),
             };
 
+            RelativeSizeAxes = Axes.Y;
+            AutoSizeAxes = Axes.X;
+
             Children = new Drawable[]
             {
                 new Box
                 {
                     RelativeSizeAxes = Axes.Both,
                     Colour = Color4.Black,
-                    Alpha = 0.8f,
+                    Alpha = 0.6f,
                 },
-                scrollContainer = new ScrollContainer
+                scrollContainer = new PaddedScrollContainer

```

Comment:
because the padding is being applied to the content specifically here. any reason you don't like this? it's the way the whole framework is made (many protected virtuals you can customise by deriving base implementations).